react/components/EditorContainer/Sidebar/BlockEditor/BlockConfigurationList/utils.ts

```diff
@@ -3,6 +3,36 @@ import { ListContentData } from '../../../queries/ListContent'
 
 import { GetDeleteStoreUpdater } from './typings'
 
+export const isConfigurationExpired = (
+  configuration: ExtensionConfiguration
+) => {
+  const parsedDate = configuration.condition.statements.reduce(
+    (acc?: Date, curr?: ExtensionConfigurationConditionStatement) => {
+      if (acc) {
+        return acc
```

Comment:
:pick: Since you're always bypassing the rest of the function once `acc` is truthy, I don't think you should be using `reduce`, because it'll always iterate over the entire array. I understand that the array is expected to be very small, but that `reduce` basically behaviors like `find`, except that `find` stops once it encounters a suitable item; therefore, `find` may be a better option.

Reply:
Great suggestion!